planning/behavior_velocity_planner/include/scene_module/scene_module_interface.hpp

```diff
@@ -147,12 +147,12 @@ class SceneModuleManagerInterface
         std::string("~/debug/path_with_lane_id/") + module_name, 1);
     }
     pub_virtual_wall_ = node.create_publisher<visualization_msgs::msg::MarkerArray>(
-      std::string("~/virtual_wall/") + module_name, 20);
+      std::string("~/virtual_wall/") + module_name, 1);
```

Comment:
This is for visualization, so it may not matter if it is large like 10.

Reply:
fixed at [41e43e4](https://github.com/autowarefoundation/autoware.universe/pull/1806/commits/41e43e481f14e119ae019059d95645f98306845f)